internal/assets/contents/shells/fishrc_append.fish

```diff
@@ -6,4 +6,8 @@ set -xg {{$K}} "{{$V}}:$PATH"
 set -xg {{$K}} "{{$V}}"
 {{- end}}
 {{- end}}
-# {{.Stop}}
\ No newline at end of file
+if test ! -z "${{.ActivatedEnv}}"
+  and test -f "${{.ActivatedEnv}}/{{.ConfigFile}}"
```

Comment:
```suggestion
if test ! -z "${{.ActivatedEnv}}" and test -f "${{.ActivatedEnv}}/{{.ConfigFile}}"
```

I was tripping over this for a sec as it wasn't obvious where the condition ended. Assuming this isn't some fish shell oddity? Man I don't like it when languages rely on line breaks for delimiters :p

Reply:
I used this reference: https://fishshell.com/docs/current/cmds/if.html#example

However, there's a `;` delimiter that acts like `and` so I'll use that.